conneg-by-ap/index.html

```diff
@@ -774,8 +774,8 @@ <h3>Functional Profiles of this specification</h3>
       implementations of systems within different environments wishing to conform to this specification.
       </p>
       <p>
-        This document provides functional profiles for two environments - HTTP &amp; human browser (Query String
-        Argument functionality) only. For the human browser environment, two functional profiles are presented.
+        This document provides functional profiles for two environments only &ndash; HTTP &amp; human browser (Query String
+        Argument functionality) only. For the human browser environment, two functional profiles are presented to cater for different modes of use.
         Further functional profiles of this specification MAY be implemented either for other environments and or even for
```

Comment:
Oh, apologies:
The word "only" occurs twice in this sentence:
"This document provides functional profiles for two environments **only** &ndash; HTTP &amp; human browser (Query String Argument functionality) **only**."
Do we really need that word? Wouldn't it suffice to say "This document provides functional profiles for two environments &ndash; HTTP &amp; human browser (Query String Argument functionality). ... Further functional profiles of this specification MAY be implemented ..."?

Reply:
Fixed as you suggest in the next commit below.